src/Modules/CalcOffence.lua

```diff
@@ -1599,6 +1601,13 @@ function calcs.offence(env, actor, activeSkill)
 				output.Speed = output.Speed * globalOutput.ActionSpeedMod
 				output.CastRate = output.Speed
 			end
+			if skillFlags.totem then
+				-- Totem skill. Apply action speed
```

Comment:
there is no source of action speed for totems afaik and tailwind do not works on them.

Reply:
> Totems still use their summoner's skills. For an offensive aura to have an effect, it must affect the summoner, not the totem. Action Speed benefits like [Tailwind](https://www.poewiki.net/wiki/Tailwind) are the one main exception and affect the totem directly.

From https://www.poewiki.net/wiki/Totem

Also @Lothrik said the same on discord.

Reply:
To be slightly more specific: the Tailwind effect from Deadeye's ascendancy node affects both the player *and nearby allies*. This means it will grant things like minions and totems action speed. Tailwind from other sources only ever affects the player. The Flow Untethered and Slipstream are two other examples of action speed sources that affect units beyond the player.